analyses/cell-type-neuroblastoma-04/run-analysis.sh

```diff
@@ -24,22 +30,40 @@ else
 fi
 
 # Define and create directories
+data_dir="../../data/current/SCPCP000004"
 script_dir="scripts"
 ref_dir="references"
 scratch_dir="scratch"
 mkdir -p $ref_dir
 mkdir -p $scratch_dir
 
-# Define NBAtlas reference object files
-# Ensure local files have the same name as on Mendeley: https://data.mendeley.com/datasets/yhcf6787yp/3
-nbatlas_subset_seurat="${scratch_dir}/SeuratObj_Share_50kSubset_NBAtlas_v20240130.rds"  # this is an older version so will need to be filtered to not contain cells missing from `nbatlas_full_seurat`
-nbatlas_full_seurat="${scratch_dir}/seuratObj_NBAtlas_share_v20241203.rds"
-nbatlas_tumor_metadata_file="${scratch_dir}/SeuratMeta_Share_TumorZoom_NBAtlas_v20250228.rds"
+set -euo pipefail
+
+# Ensure script is being run from its directory
+module_dir=$(dirname "${BASH_SOURCE[0]}")
+cd ${module_dir}
 
-nbatlas_subset_sce="${ref_dir}/NBAtlas_subset_sce.rds"
-nbatlas_subset_anndata="${ref_dir}/NBAtlas_subset_anndata.h5ad"
-nbatlas_full_sce="${ref_dir}/NBAtlas_full_sce.rds"
-nbatlas_full_anndata="${ref_dir}/NBAtlas_full_anndata.h5ad"
+# Set up the testing flag
+testing=${testing:-0}
+if [[ $testing -eq 1 ]]; then
+    test_flag="--testing"
+else
+    test_flag=""
+fi
+
+# Define and create directories
+script_dir="scripts"
+ref_dir="references"
+scratch_dir="scratch"
+mkdir -p $ref_dir
+mkdir -p $scratch_dir
+
+nbatlas_sce="${ref_dir}/NBAtlas_${nbatlas_version}_sce.rds"
+nbatlas_anndata="${ref_dir}/NBAtlas_${nbatlas_version}_anndata.h5ad"
```

Comment:
So... if we aren't using the AnnData anymore, why don't we just get rid of this and any of the script that performs the conversion?

Reply:
Wait we are using this AnnData since we will need it for scArches. I think I would keep it in and not remove converting it as you did later on.

Reply:
I can definitely revert this, but to clarify I would plan to turn it back on once we get to scArches! It just takes a decent amount of time in CI, and we don't need that AnnData version yet. I could also open an issue to remind us to revert it back once we get to that stage? Or, I'll revert it now if you really think I should. :)